lib/pages/detail/detail_content.dart

```diff
@@ -18,128 +19,31 @@ class DetailContent extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    if (searchResult.companies == null) {
+    if (searchResult.companies == null || searchResult.companies!.isEmpty) {
       return Padding(
         padding: const EdgeInsets.all(8.0),
         child: Text(searchResult.altText ?? ""),
       );
     }
 
     final company = searchResult.companies!.first;
-    final score = company.plScore ?? 0;
-    final double plCapital = (company.plCapital ?? 0).toDouble();
-    final Translations t = Translations.of(context);
+    (company.plCapital ?? 0).toDouble();
+    Translations.of(context);
```

Comment:
Co robią te linie? Jak analyzer mówi że coś jest nieużywane to najpewniej cała linia leci do kosza. Tu zostały jakieś dziwne obliczenia które nigdzie nie są przypisane i sobie wiszą bez sensu